salt/modules/runit.py

```diff
@@ -164,6 +164,85 @@ def available(name):
     return name in get_all()
 
 
+def enabled(name, **kwargs):
```

Comment:
We prefer that these `**kwargs` be removed. The interface definitions for execution modules should be as well-defined as possible. This also helps with our routines which attempt to parse the argspecs in the case of errors.